python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_ishigami.py

```diff
@@ -220,6 +259,25 @@ def ishigamiSA(a, b):
         % (i + 1, absoluteErrorS, absoluteErrorST)
     )
 
-plt.show()
 # %%
 # We see that the indices are correctly estimated with a low accuracy even if we have use only 100 function evaluations. This shows the good performance of the polynomial chaos in this case.
+
+# %%
+# We can compute the part of variance of each multi-index.
```

Comment:
Could we maybe say "the part of the variance of the output explained by each multi-index"?

Reply:
Ok. Fixed at 30c0beb4dc63400c081acc1010ae8152e820004f.